packages/block-editor/src/store/actions.js

```diff
@@ -1214,7 +1217,6 @@ export function replaceInnerBlocks(
 		blocks,
 		updateSelection,
 		initialPosition: updateSelection ? initialPosition : null,
-		time: Date.now(),
```

Comment:
Can we also remove `time` from the `INSERT_BLOCKS` action object?

Reply:
Yeah, that's curious. Both `REPLACE_BLOCKS` and `INSERT_BLOCKS`. I wonder why `REPLACE_INNER_BLOCKS` has it anyway, as it doesn't update insert usage.

I'll push a commit that removes it entirely.